modules/gdnative/pluginscript/pluginscript_instance.cpp

```diff
@@ -0,0 +1,151 @@
+// Godot imports
+#include "core/os/os.h"
+#include "core/variant.h"
+// PluginScript imports
+#include "pluginscript_instance.h"
+#include "pluginscript_language.h"
+#include "pluginscript_script.h"
+
+bool PluginScriptInstance::set(const StringName &p_name, const Variant &p_value) {
+	String name = String(p_name);
+	return this->_desc->set_prop(this->_data, (const godot_string *)&name, (const godot_variant *)&p_value);
```

Comment:
As you may have noticed, I'm a python guy so I love to access my instance's attribute explicitly (e.g. this->_attribute) ^^

This is not what is done in the rest of the project so I can update this to a more traditional implicit this, just tell me if it hurts you ;p

Reply:
It doesn't exactly *hurt* me, but I'd still prefer to remove the `this` pointers. ^^

Reply:
Yeah, if only for consistency, it's better not to use `this` pointers when not necessary.

Reply:
If you prefer I can also provide a PR to put `this->` everywhere in Godot codebase :trollface: